docs/source/developer/contributing.rst

```diff
@@ -908,17 +908,17 @@ Or with ``jlpm``:
 Writing Style
 ^^^^^^^^^^^^^
 
--  The documentation should be written in the second person, referring
-   to the reader as "you" and not using the first person plural "we."
+-  Write documentation in the second person, referring
+   to the reader as "you". Do not use the first person plural "we."
    The author of the documentation is not sitting next to the user, so
    using "we" can lead to frustration when things don't work as
    expected.
 -  Avoid words that trivialize using JupyterLab such as "simply" or
    "just." Tasks that developers find simple or easy may not be for
    users.
--  Write in the active tense, so "drag the notebook cells..." rather
-   than "notebook cells can be dragged..."
--  The beginning of each section should begin with a short (1-2
+-  Write in the active tense. Use "drag the notebook cells…" rather
+   than "notebook cells can be dragged…".
+-  The beginning of each section should begin with a short (1–2
```

Comment:
Seems like an unintended long dash was added here instead of a hyphen.

Reply:
That's an [en dash](https://en.wikipedia.org/wiki/Dash#En_dash), which indicates a range of values among other purposes.